lib/include/ert/ecl/ecl_file.h

```diff
@@ -38,7 +38,8 @@ extern "C" {
 
 #define ECL_FILE_FLAGS_ENUM_DEFS \
   {.value =   1 , .name="ECL_FILE_CLOSE_STREAM"}, \
-  {.value =   2 , .name="ECL_FILE_WRITABLE"}
+  {.value =   2 , .name="ECL_FILE_WRITABLE"}, \
+  {.value =   4 , .name="ECL_FILE_RETURN_COPY"}.
 #define ECL_FILE_FLAGS_ENUM_SIZE 2
```

Comment:
should this be 3?

Reply:
No - these values are used as bitmasks, so it should indeed be 4 - or 8, or ...

Reply:
Joakim, I'm thinking about this:
```c
#define ECL_FILE_FLAGS_ENUM_SIZE 2
```

Reply:
Ahhh yes - that is right :-)